packages/core/src/builder.class.ts

```diff
@@ -422,6 +422,7 @@ export class Builder {
   authToken = '';
 
   static editors: any[] = [];
+  static trustedHosts: string[] = ['builder.io', 'localhost:1234'];
```

Comment:
```suggestion
  static trustedHosts: string[] = ['builder.io', 'localhost'];
```
this check is against `hostname` which excludes the port (ant not `host` which includes it)